ui/src/pages/frontsite/StaticPage/StaticPage.js

```diff
@@ -0,0 +1,49 @@
+import { Error } from '@components';
+import {
+  ILSHeaderPlaceholder,
+  ILSParagraphPlaceholder,
+} from '@components/ILSPlaceholder';
+import { getStaticPageByRoute } from '@config/uiConfig';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Container, Header } from 'semantic-ui-react';
+import _isEmpty from 'lodash/isEmpty';
+
+export default class StaticPage extends Component {
+  componentDidMount() {
+    const staticPage = getStaticPageByRoute(this.props.match.path);
+    const staticPageID = staticPage['apiURL'];
```

Comment:
if it is not configured it will go to 404, because of how I wrote the routing based on the config. If there will be a route in `match.path` that is not coming from the static pages configuration, it will not even arrive to this component. Checking all the paths against the config to throw the error seems to me a bit complicated in comparison to what we gain, unless you have an idea ?